src/shapes/group.class.js

```diff
@@ -83,25 +96,32 @@
 
     /**
      * @private
+     * @param {Boolean} [unchangeCoords] if true, coordinates of objects enclosed in a group do not change
      */
-    _updateObjectsCoords: function() {
-      this.forEachObject(this._updateObjectCoords, this);
+    _updateObjectsCoords: function(unchangeCoords) {
+      for (var i = this._objects.length; i--; ){
+        this._updateObjectCoords(this._objects[i], unchangeCoords);
+      }
     },
 
     /**
      * @private
-     */
-    _updateObjectCoords: function(object) {
-      var objectLeft = object.getLeft(),
-          objectTop = object.getTop(),
-          center = this.getCenterPoint();
-
-      object.set({
-        originalLeft: objectLeft,
-        originalTop: objectTop,
-        left: objectLeft - center.x,
-        top: objectTop - center.y
-      });
+     * @param {Object} object
+     * @param {Boolean} [unchangeCoords] if true, coordinates of object dose not change
+     */
+    _updateObjectCoords: function(object, unchangeCoords) {
+      if (!unchangeCoords) {
```

Comment:
i think means that 
```diff
_updateObjectCoords: function(object) {
// this always:
object.__origHasControls = object.hasControls;
object.hasControls = false;
 if (skipCoordsChange) {
  //nothing else...
   return;
 }

   var center = this.getCenterPoint();

    object.set({
      originalLeft:  object.getLeft(),
      originalTop: object.getTop(),
      left:  object.getLeft() - center.x,
      top: object.getTop() - center.y
    });
    //we do not setCoords if we do not update top and left.
    object.setCoords();
}
```

sorry for messing up indentation.
We are looking forward release 1.5.1 and this PR is a milestone!